Content.Server/Construction/ConstructionSystem.Initial.cs

```diff
@@ -121,6 +125,15 @@ private IEnumerable<EntityUid> EnumerateNearby(EntityUid user)
 
                 yield return item;
             }
+            // <Goobstation> - lets slimepeople and constructors use their storage
+            if (TryComp<StorageComponent>(user, out var userStorage))
+            {
+                foreach (var userItem in userStorage.Container.ContainedEntities!)
+                {
+                    yield return userItem;
+                }
+            }
```

Comment:
cursed 1 line statements be upon ye
```suggestion
            if (TryComp<StorageComponent>(user, out var userStorage))
                foreach (var userItem in userStorage.Container.ContainedEntities!)
                    yield return userItem;

Reply:
no this is evil

Reply:
evilstation